src/deadline/keyshot_submitter/Submit to AWS Deadline Cloud.py

```diff
@@ -370,54 +436,62 @@ def main(lux):
         else:
             lux.saveFile()
 
-    scene_file = lux.getSceneInfo()["file"]
-    external_files = lux.getExternalFiles()
+    dialog_selections = options_dialog()
+
+    if not dialog_selections:
+        print("Job submission canceled.")
+        return
+
+    scene_info = lux.getSceneInfo()
+    scene_file = scene_info["file"]
+    scene_name, _ = os.path.splitext(scene_info["name"])
     current_frame = lux.getAnimationFrame()
     frame_count = lux.getAnimationInfo().get("frames")
 
     settings = Settings(
         parameter_values=[
-            {
-                "name": "KeyShotFile",
-                "value": scene_file,
-            },
             {
                 "name": "Frames",
                 "value": f"1-{frame_count}" if frame_count else f"{current_frame}",
             },
             {
                 "name": "OutputFilePath",
-                "value": f"{scene_file}.%d.png",
+                "value": os.path.join(os.path.dirname(scene_file), f"{scene_name}.%d.png"),
             },
             {
                 "name": "OutputFormat",
                 "value": "PNG",
             },
         ],
         input_filenames=[],
-        auto_detected_input_filenames=[*external_files, scene_file],
+        auto_detected_input_filenames=[],
         input_directories=[],
         output_directories=[],
-        auto_detected_output_directories=[str(os.path.dirname(scene_file))],
         referenced_paths=[],
     )
 
-    _, filename = os.path.split(scene_file)
-
     sticky_settings = load_sticky_settings(scene_file)
     if sticky_settings:
         settings.apply_sticky_settings(sticky_settings)
 
-    job_template = construct_job_template(filename)
-    asset_references = construct_asset_references(settings)
-    parameter_values = construct_parameter_values(settings)
+    with tempfile.TemporaryDirectory() as bundle_temp_dir:
+        # {'submission_mode': [0, 'the scene BIP file and all external files references']}
+        if not dialog_selections[SUBMISSION_MODE_KEY][0]:
+            temp_scene_file, input_filenames = get_ksp_bundle_files(bundle_temp_dir)
+            settings.auto_detected_input_filenames = input_filenames
+            settings.parameter_values.append({"name": "KeyShotFile", "value": temp_scene_file})
+        else:
```

Comment:
Do we need to attach the scene file in this branch? Also do we still want to attach external files if KSP bundling isn't selected? (I think not, but just double checking.)

Reply:
The scene file parameter is a PATH parameter, just by having it specified it will include it as an input file, no need to actually add it to the asset references directly

Reply:
No, if just the .bip file is selected we don't want to attach any of the external files since they should be made available through network storage or some other means.